INSTALL.md

```diff
@@ -26,3 +26,15 @@ example:
 make PREFIX=/usr SYSCONFDIR=/etc LOCALSTATEDIR=/var
 make PREFIX=/usr SYSCONFDIR=/etc LOCALSTATEDIR=/var DESTDIR=/tmp/rpmbuildroot install
 ```
+
+# Detailed steps how to install on RHEL 7 machine
```

Comment:
Rather than carrying specific distribution instructions in the project repository, I'd rather keep a list of prerequisite software that may need to be installed in order to build the project. For example:

```
# Prerequisites

Some additional software may be required in order to compile yggdrasil.

For RPM-based systems:
* dbus-devel
```